docs/additional-functionality/qualification-profiling-tools.md

```diff
@@ -224,6 +224,15 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
                                     filesystem timestamp). 100-oldest-filesystem
                                     (for processing oldest 100 event logsbased
                                     on filesystem timestamp).
+  -l, --logic-filter <arg>          Filter event logs where any or all filter criteria
```

Comment:
Can we just call these options `--any` and `--all`?  `logic-filter` is likely to confuse users since it is unidiomatic and also not exactly correct (it's not a logic filter; it's an operation to combine several filters).  It's also a lot of keystrokes.

Reply:
Thanks @willb for your feedback. Updated the code.